File: presentation/src/main/java/com/mbmc/template/di/ActivityModule.java
package com.mbmc.template.di;

import com.mbmc.template.ui.MainActivity;
import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class ActivityModule {

    @ContributesAndroidInjector
    abstract MainActivity contributeActivityInjector();

}

File: presentation/src/main/kotlin/com/mbmc/template/di/module/ActivityModule.kt
package com.mbmc.template.di.module

import com.mbmc.template.ui.MainActivity
import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class ActivityModule {
    @ContributesAndroidInjector
    abstract fun contributeActivityInjector(): MainActivity
}
